src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFADetailsPage.java

```diff
@@ -44,4 +44,11 @@ public String getNewAccountSavedName() {
         return CommonActions.getElement(this.newAccountLabel).getText();
     }
 
+    /**
+     * Wait until the page loads.
+     */
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)